internal/store/postgres/resource/repository.go

```diff
@@ -53,27 +53,81 @@ func (r Repository) Update(ctx context.Context, resourceModel *resource.Resource
 	return nil
 }
 
+func (r Repository) Delete(ctx context.Context, resourceModel *resource.Resource) error {
+	res := FromResourceToModel(resourceModel)
+
+	deleteResourceQuery := `UPDATE resource SET status=$5, updated_at=NOW()
```

Comment:
any reason why we start with $5? not a problem, but might not convenient to read unordered.

Reply:
there's no particular reason, its just i copied where clause query from `ReadByFullName`